shell/components/xen/xen-element.js

```diff
@@ -70,8 +70,10 @@ export default Base => class extends Base {
   }
   _didMount() {
   }
-  _fire(eventName, detail, node) {
-    const event = new CustomEvent(eventName, {detail});
+  _fire(eventName, detail, node, init) {
```

Comment:
Just curious, is this planning ahead, or does something already use this and it's not working without this change?

Reply:
I have a shell prototype (vis a vis Yuin's designs) that uses this feature, otherwise it's not used today.